src/legend/svgLegend.ts

```diff
@@ -357,35 +364,42 @@ export class SVGLegend implements ILegend {
 
         legendItems
             .select(SVGLegend.LegendIcon.selectorName)
-            .attr({
-                "cx": (d: LegendDataPoint, i) => d.glyphPosition.x,
-                "cy": (d: LegendDataPoint) => d.glyphPosition.y,
-                "r": iconRadius,
-            })
-            .style({
-                "fill": (d: LegendDataPoint) => {
+            .attr(
+                "cx", (d: LegendDataPoint, i) => d.glyphPosition.x
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line